### PR TITLE
Added contact form to profile

### PR DESCRIPTION
**Pull request by emyrue**
- Added form
- Added input fields for name and email
- Added a text area
- Added social media icons

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,13 @@ As you work your way through Microverse curriculum materials you will complete s
...
+- GitHub: [@emyrue](https://github.com/emyrue)
+- Twitter: [@EmyrueRobertson](https://twitter.com/EmyrueRobertson)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/emily-robertson-70a2bb22a/)
 
 👤 **Author2**
 
-- GitHub: [@githubhandle]()
-- Twitter: [@twitterhandle]()
-- LinkedIn: [LinkedIn]()
+- GitHub: [[@dewslyse](https://github.com/dewslyse)
```
Comment (Donard97) [inline README.md:27]: - [ ] You did a great job adding your partner on the README file but please delete that extra "[" that is before GitHub username. Keep coding 💪

File: index.html [hunk 2]
```diff
@@ -1,206 +1,277 @@
...
-                </ul>
-            </section>
+    <!--Form-->
+
+    <section class="contact-form">
+        <img alt="Logo" class="geometry" src="./images/image_geometry_1.png">
+        <h2 class="contact-heading">Get started</h2>
+        <p class="contact-paragraph">Lorem ipsum dolor sit amet,
+            consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt
```
Comment (Donard97) [inline index.html:212]: - [ ] Good job on adding contact form section but kindly wrap it inside of footer as it was part of [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). Keep rocking 💪

File: index.html [hunk 3]
```diff
@@ -1,206 +1,277 @@
...
+                maxlength="500"></textarea>
+            <button class="form-button" type="submit">Start Collaboration</button>
+        </form>
+    </section>
+
+    <!--Socials-->
+
+    <section class="socials">
+        <section class="resume">
+            <h2 class="get-resume">Get my resume</h2>
```
Comment (Donard97) [inline index.html:233]: - [ ] I really like the work that you did on adding the socials section at the end of the page but I suggest that you delete the socials section and put icons inside of the footer.

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hi @emyrue , 👋  

You did a tremendous job so far! 💯 💯 💯 

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)


## Required Changes ♻️
- [ ]  Please check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @emyrue & @dewslyse , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: Well implemented of required changes.

### Optional suggestions :arrow_down_small: 

- [ ] Kindly use the imperative mood in the subject line for commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**